common/src/main/java/io/homeassistant/companion/android/common/sensors/SensorReceiverBase.kt

```diff
@@ -320,14 +330,12 @@ abstract class SensorReceiverBase : BroadcastReceiver() {
         integrationUseCase.registerSensor(reg)
     }
 
-    suspend fun updateSensor(
-        context: Context,
+    private suspend fun updateSensor(
         integrationUseCase: IntegrationRepository,
         fullSensor: SensorWithAttributes?,
-        sensorManager: SensorManager?,
-        basicSensor: SensorManager.BasicSensor
+        basicSensor: SensorManager.BasicSensor,
+        sensorDao: SensorDao
     ) {
-        sensorManager?.requestSensorUpdate(context)
```

Comment:
Why move updating the sensor out of the `updateSensor` function?

Reply:
I was not seeing the correct approach earlier 🙈